src/System.Private.ServiceModel/tests/Common/Scenarios/project.json

```diff
@@ -1,10 +1,14 @@
 {
   "dependencies": {
     "Microsoft.NETCore.Platforms": "1.0.1-rc3-23809",
-    "System.Collections.Concurrent": "4.0.0",
-    "System.Net.Http": "4.0.1-rc3-23809",
+	"System.Collections.Concurrent": "4.0.0",
+	"System.ComponentModel.EventBasedAsync": "4.0.10",
+	"System.Net.Http": "4.0.1-rc3-23809",
```

Comment:
I saw you changed S.P.SM to need 4.0.0 of S.N.Http. Why does the test need 4.0.1-rc3-xxxxx here?